cmd/podman/runlabel.go

```diff
@@ -146,10 +147,26 @@ func runlabelCmd(c *cliconfig.RunlabelValues) error {
 		return err
 	}
 	if !c.Quiet {
-		fmt.Printf("Command: %s\n", strings.Join(cmd, " "))
+		fmt.Printf("command: %s\n", strings.Join(cmd, " "))
 		if c.Display {
 			return nil
 		}
 	}
+
+	// If container already exists && --replace given -- Nuke it
+	if c.Replace {
+		for i, entry := range cmd {
```

Comment:
food for thought, would shlex.split and index func avoid some of this nesting?  again, just a thought

Reply:
ok it is already split ...